src/redist/targets/BundledTemplates.targets

```diff
@@ -91,28 +91,32 @@
 
     <PackageDownload Include="@(Bundled21Templates)" Exclude="@(PackageDownload)" />
     <PackageDownload Include="@(Bundled22Templates)" Exclude="@(PackageDownload)" />
+    <PackageDownload Include="@(Bundled30Templates)" Exclude="@(PackageDownload)" />
 
     <PackageDownload Update="Microsoft.DotNet.Common.ItemTemplates"
                      Version="[$(MicrosoftDotNetCommonItemTemplates21PackageVersion)];
                               [$(MicrosoftDotNetCommonItemTemplates22PackageVersion)];
+                              [$(MicrosoftDotNetCommonItemTemplates30PackageVersion)];
                               " />
 
-    <PackageDownload Update="Microsoft.DotNet.Web.ItemTemplates;Microsoft.DotNet.Web.Spa.ProjectTemplates"
+    <PackageDownload Update="Microsoft.DotNet.Web.ItemTemplates"
+                     Version="[$(AspNetCorePackageVersionFor21Templates)];
+                              [$(AspNetCorePackageVersionFor22Templates)];
+                              [$(AspNetCorePackageVersionFor30Templates)];
+                              " />
+
+    <PackageDownload Update="Microsoft.DotNet.Web.Spa.ProjectTemplates"
```

Comment:
No 3.0 for this one?

Reply:
The 3.0 package has a unique name and doesn't need this.